governance-aepp/src/components/PollListing.vue

```diff
@@ -46,6 +44,10 @@
         height: 0
       };
     },
+    setup() {
+      const {networkId} = toRefs(wallet)
+      return {networkId}
+    },
```

Comment:
afaik this is only needed when you use the networkId in the template

Reply:
I want to use it elsewhere with `this.` seems to work as is